unit-tests/src/org/commcare/android/tests/caselist/EntityListCalloutDataTest.java

```diff
@@ -75,6 +75,7 @@ public void testAttachCalloutResultToListTest() {
 
         // ensure that the entity list is filtered by the received callout
         // result data (fingerprint identification list with confidence score)
+        loadList();
```

Comment:
yeah, happy to create a new PR if you would like. I just noticed random failing tests and wanted to try fixing them.